### PR TITLE
(#1666) Font Family Utility Page

### LINKED ISSUE #1666: Update documentation site Font Size & Family page content to use Utility page template
### Request Description
This change request is update the Font Size & Family page on the NCIDS from only having basic information about font tokens/utilities to using our Utility page template.

- [ ] IA/Design updates to the [Font Size & Family doc site page](https://designsystem-dev.cancer.gov/develop/foundations/typography/fonts) with full instructions on use
- [ ] Design updates to the [community Figma](https://www.figma.com/file/4FeMfprVErO6esR8FLczp5/NCIDS-Design-Kit?type=design&node-id=0-1&mode=design&t=Ec8CKPLUg3f7Jlm4-0)
- [ ] Developer updates to the code to adopt Font Size & Family tokens/utilities within the NCIDS

### Important Notes
* The content on this page will use the utility page template outlined in #1593.
* [Utilities spreadsheet](https://nih.sharepoint.com/:x:/r/sites/NCI-OCPLNewCgovUXDesignSystem/Shared%20Documents/General/Details%20-%20Release%202/R2.4/Development%20and%20Testing/USWDS_Utilities.xlsx?d=waeac153a696a4419941b142cc9b627f1&csf=1&web=1&e=8THgYK)
* [USWDS Font Size & Family page](https://designsystem.digital.gov/utilities/font-size-and-family/)

### What's the expected change?
- The Doc Site is updated to show users how to fully use font size & family tokens/utilities on the NCIDS Doc Site without pointing to USWDS
- The community Figma file will be updated to include font size & family tokens/utilities 
- The NCIDS code will be updated to fully adopt font size & family tokens/utilities 

### What's the current functionality?
- The NCIDS has some information on font size & family tokens.

### Related Tickets
* #1593

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1666

## Pull Request Details

_Add description_

Closes #<TICKET-NUMBER>

## Author PR Checklist
Items that the author of the PR is responsible for checking before submitted the PR.

### General:
- [x] I have reviewed the acceptance criteria defined in the ticket and ensured the work has been completed.
- [x] The commit message passes [all quality commit message standards](https://github.com/NCIOCPL/ncids/wiki/Creating-Quality-Commit-Messages).
- [x] Unit tests have been updated or created to reflect any javascript changes.
- [x] Storybook scenarios have been updated or created to reflect any html/css/js changes.

### Accessibility:
- [x] WCAG 2.1 Level AA requirements have been met.

### Development:
- [x] Any new or updated javascript code has 100% unit test coverage.
- [x] New or updated breakpoints have regression images.
- [x] Breaking changes have been thoroughly documented in the PR.

## Product Reviewer PR Checklist
Items the product team is responsible for reviewing.

### General:
- [x] There are no unexpected or unapproved regression image changes. 
- [x] Functionality of interactive elements meet the acceptance criteria.
- [x] The product is visually and functionally the same across the different browsers.

### Accessibility:
- [x] AxeDev Tools: there are no new or outstanding accessibility issues introduced in this PR.
- [x] Lighthouse: scores have not noticeably decreased during this PR.
- [x] Wave: there are no new errors or contrast errors introduced in this PR.

## Design Reviewer PR Checklist
Items the design team is responsible for reviewing.  

### General:
- [x] New or updated features introduced in this PR are developed mobile-first.
- [x] Breakpoint changes and regression images match those breakpoints.
- [x] This PR has been tested in all supported browsers at all breakpoints.

## Developer Reviewer PR Checklist
Items the development team is responsible for reviewing.

### General:
- [x] New code passes code quality standards set by industry standards.
- [x] The expected Storybook stories have been added or updated for the new or updated feature.
- [x] The expected unit tests have been added or updated for the new or updated feature.

### Accessibility:
- [x] VoiceOver: Described content matches with what was expected.
- [x] Keyboard navigation: new or updated features and content are navigable via the keyboard.

### CODE REVIEW

File: docs/content/foundations/typography/fonts.mdx [hunk 1]
```diff
@@ -102,157 +195,3 @@ There are two types of size tokens: `system` and `theme` tokens.
...
-	<FontSizeThemeRow token="2xs" size="3" variable="theme-type-scale-2xs" />
-	<FontSizeThemeRow token="xs" size="5" variable="theme-type-scale-xs" />
-	<FontSizeThemeRow token="sm" size="7" variable="theme-type-scale-sm" />
-	<FontSizeThemeRow token="md" size="8" variable="theme-type-scale-md" />
-	<FontSizeThemeRow token="lg" size="9" variable="theme-type-scale-lg" />
-	<FontSizeThemeRow token="xl" size="10" variable="theme-type-scale-xl" />
-	<FontSizeThemeRow token="2xl" size="11" variable="theme-type-scale-2xl" />
-	<FontSizeThemeRow token="3xl" size="12" variable="theme-type-scale-3xl" />
-</FontSizeThemeTable>
-
```
Comment (bryanpizzillo) [inline docs/content/foundations/typography/fonts.mdx:249]: This should be kept. This is showing the scale, which is related to the tokens

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
I have one change request inline. I also have a question -- is there any way we can mimic USWDS and how they show the fonts? It is nice seeing the differences stacked next to each other, without additional text in between.

**Review by bryanpizzillo — APPROVED** [review]
This is looking pretty good now.

**Review by sarinapadilla — APPROVED** [review]
Looks good to product!

**Pull request merged**